scripts/postdeploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd app
+
+# Run Django migrations.
+python manage.py migrate
+# Seed database with dummy users
+python manage.py seedusers
```

Comment:
we can do that in postdeploy can't we? https://github.com/mozilla/api.mozillascience.org/pull/85/files#diff-a3d148fd9118e9bc0c1b77c41d191081R7